config/formats.js

```diff
@@ -839,21 +839,19 @@ let Formats = [
 		column: 2,
 	},
 	{
-		name: "[Gen 7 Pet Mod] Clean Slate: Micro",
-		desc: `A brand new "micrometagame" created from scratch, with the ultimate goal of creating a unique, compact metagame different from any other tier.`,
+		name: "[Gen 8] Roulettemons",
+		desc: `A competitive metagame made up of brand new Pokémon that are randomly generated on every competitive aspect (typing, moves, stats, abilities).`,
 		threads: [
-			`<a href="https://www.smogon.com/forums/threads/3652540/">Clean Slate: Micro</a>`,
+			`<a href=https://www.smogon.com/forums/threads/.3649106/>Roulettemons</a>`,
 		],
-
-		mod: 'cleanslatemicro',
-		ruleset: ['Standard Pet Mod'],
+		mod: 'roulettemons',
+		ruleset: [ 'Dynamax Clause', 'Obtainable Moves', 'Sleep Clause Mod', 'Species Clause', 'Moody Clause', 'Evasion Moves Clause', 'Endless Battle Clause', 'HP Percentage Mod', 'Cancel Mod', 'Team Preview', 'Swagger Clause', 'Baton Pass Clause'],
+		banlist: ['All Pokemon'],
```

Comment:
```suggestion
		mod: 'roulettemons',
		ruleset: ['Standard Pet Mod'],
```

The point of Standard Pet Mod is to put all of the custom rules and bans in the ruleset.